omero/omero-firewall.yml

```diff
@@ -0,0 +1,61 @@
+# Setup up iptables firewall on OMERO servers
+
+- hosts: monitored
```

Comment:
Discussed with @kennethgillen used of this group is inaccurate named since this playbook is related to firewalling not monitoring, but for convenience we'll keep this until it's rolled out to all relevant production systems

Reply:
👍